src/Components/Detail/SideBar.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components/macro';
+import { flexCenter } from '../../style/mixin';
+
+export default function Detail() {
+  const [isLiked, setIsLiked] = useState(false);
+  const toggleHeart = () => {
+    setIsLiked(!isLiked);
+  };
+  return (
```

Comment:
```suggestion
export default function Detail() {
  const [isLiked, setIsLiked] = useState(false);
  
  const toggleHeart = () => {
    setIsLiked(!isLiked);
  };
  
  return (
```
줄바꿈을 통해 가독성을 올려볼까요~?